indico/modules/events/registration/controllers/management/regforms.py

```diff
@@ -47,6 +48,31 @@ def _process(self):
                                                     event=self.event, regforms=regforms)
 
 
+class RHManageParticipants(RHManageRegFormsBase):
+    """Show and enable the dummy registration form for participants"""
+
+    def _process_POST(self):
+        regform = RegistrationForm.find_first(event_id=self.event.id, is_participation=True, is_deleted=False)
```

Comment:
`self.event_new.registration_forms.filter_by(is_participation=True, is_deleted=False).first()` would also work

Could even be a property on `Event`